openupgradelib/openupgrade.py

```diff
@@ -3492,3 +3493,32 @@ def cow_templates_replicate_upstream(cr, mark_colname=None):
         """
         ).format(mark_identifier),
     )
+
+
+def clean_transient_models(cr):
+    """Clean transient models to prevent possible issues due to
+    chained data.
+
+    To be run at the base post-migration script for having a general scope.
```

Comment:
Now this can be run on pre-migration:

```suggestion
    To be run at the base pre-migration script for having a general scope.
```